postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -100,7 +100,7 @@
     """
     SELECT application_name, state, usename, datname, count(*) as connections
     FROM {pg_stat_activity_view}
-    WHERE client_port IS NOT NULL
+    WHERE pid != pg_backend_pid() AND client_port IS NOT NULL
```

Comment:
shouldn't we still count the agent's connection as an active one to the db? this is just for the metric

Reply:
I feel like we should _not_... otherwise it'll suffer the same issue, i.e. that metric will always report 1 on a lone instance.

Reply:
But happy to discuss if you disagree